utils.go

```diff
@@ -197,9 +198,9 @@ func parseMsg(msg *pb.WebcastResponse_Message, warnHandler func(...interface{}),
 						ID:       int64(user.Id),
 						Username: user.ProfileId,
 						Nickname: user.Name,
-						ProfilePicture: &ProfilePicture{
-							Urls: urls,
-						},
+						// ProfilePicture: &ProfilePicture{
```

Comment:
Either delete this comment or fix it so we have a new host type.
I am ok with just deleting these comments and making a change to add a Host type later.